pages/game.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+
+import ActivityIndicator from '../src/components/activityIndicator';
+import Bloc from '../src/components/bloc';
+import Button from '../src/components/button';
+import Grid from '../src/components/grid';
+import Page from '../src/components/page';
+import Row from '../src/components/row';
+import Section from '../src/components/section';
+
+import { buildGrid } from '../src/core/game';
+import { shuffle } from '../src/core/shuffler';
+
+export default class Index extends Component {
+    state = {
+        isLoading: true,
+        currentGrid: null,
+        initialGrid: null,
+        turn: -1,
+        isVictory: false,
+    };
+
+    handleClick = () => {};
+
+    renderWinnerMessage(isVictory, turn) {
```

Comment:
Having functions like this is usually a sign that the component should be split. That or use an inline if.